packages/react-instantsearch-hooks/README.md

```diff
@@ -468,6 +468,133 @@ function Hits(props) {
 }
 ```
 
+
+### `useMenu`
+
+> `(props: UseMenuProps) => MenuRenderState`
+
+Hook to use a [menu](https://www.algolia.com/doc/api-reference/widgets/menu/js/).
+
+**Types**
+
+<details>
+<summary><code>UseMenuProps</code></summary>
+
+```ts
+type UseMenuProps = {
+  /**
+   * The name of the attribute in the records.
+   */
+  attribute: string;
+  /**
+   * The max number of items to display when
+   * `showMoreLimit` is not set or if the widget is showing less value.
+   */
+  limit?: number;
```

Comment:
In other hooks, we've documented the default values for optional arguments with the [`@default`](https://jsdoc.app/tags-default.html) directive, at least for primitives (see [`useHits`](https://github.com/algolia/react-instantsearch/tree/feat/hooks-menu/packages/react-instantsearch-hooks#usehits)).